src/sample/signUpController.java

```diff
@@ -51,122 +51,127 @@ public void openBrowser() throws URISyntaxException, IOException {
   }
 
   public void signUp(MouseEvent event) throws IOException {
-    if (id.getText().isBlank() || password.getText().isBlank() || rePassword.getText().isBlank()) {
-      signUpMessage.setText("Please fill in all required fields!");
-    } else if (id.getLength() != 6) {
-      signUpMessage.setText("ID must be 6 characters");
-    } else if (!password.getText().equals(rePassword.getText())) {
-      signUpMessage.setText("Password does not match!");
-    } else if (password.getText().equals(rePassword.getText()) && password.getLength() < 6) {
-      signUpMessage.setText("Password must have minimum 6 characters!");
-    } else{
-      if(rbStudent.isSelected()){
-        File file1 = new File("data/Student Profile/"+id.getText()+".txt");
-        File file2 = new File("data/Student Course List/"+id.getText()+".txt");
-        File file3 = new File("data/Student Registered Course/"+id.getText()+".txt");
-      
-        //Check whether if the account exist or not
-        if(!file1.exists() && !file2.exists() && !file3.exists()){
-          try{
-            //Create Student Profile txt file and save their matric number and password into it
-            file1.createNewFile();
-            BufferedWriter writer=new BufferedWriter(new FileWriter(file1));
-            writer.write(id.getText() + "\n");
-            writer.write(password.getText() + "\n");
-            writer.close();
-          
-            //Create Student Course List txt file and copy the Course List into it
-            file2.createNewFile();
+    try {
```

Comment:
Why this file feels a bit old one? Just to double confirm, did you pull the latest version?